.github/workflows/wheels.yml

```diff
@@ -26,12 +26,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest] #, macos-latest]
 
     steps:
     - uses: actions/checkout@v1
       with:
-        submodules: true
+        submodules: recursive
+
+    - name: Install extra deps on Linux
+      if: runner.os == 'Linux'
+      run: sudo apt-get update && sudo apt-get install -y libmpfr-dev libboost-dev
```

Comment:
No, the wheels are not built here. They are built in a docker container. You need to set:

```
CIBW_BEFORE_ALL_LINUX: "yum update && yum install -y libmpfr-devel libboost-devel"
```

(or whatever these are called in CentOS 5 or 6, depending on if you are using manylinux1 or 2010). For macOS, you'll need to use brew instead.

Reply:
You might like the new pyproject.toml configuration in cibuildwheel 2.0.0.a4 instead, the final version will likely be out in a few weeks. That would be:

```toml
[tool.cibuildwheel.linux]
before-all = [
    "yum update",
    "yum install -y libmpfr-devel libboost-devel",
]
```

Reply:
Thanks for the help! We've turned off MacOS in the wheels.yml because it's not being tested in ci.yml. We intend to turn that back on someday, but the current effort is for getting a usable first version as a release candidate on PyPI.

Reply:
Got it.  Is the command in the right place now? @henryiii

Reply:
Yes. :)

Reply:
I have corrected the names for CentOS, and the dependencies were installed as well (as per the log). What if the Fortran compiler is also an issue here? Because last time adding these dependencies worked for CGAL in Ubutnu. @henryiii

Reply:
I'm tempted to move this to cibuildwheel 2.0.0a4 and use pyproject.toml - then we can very simply add a test job building a single wheel, like I did for awkward (though we can do it already if we don't mind duplicating the setup variables - but not ideal). To test this currently you'll need to manually trigger it in the GitHub UI.

As long as you don't link to anything you can't bundle that's not in the manylinux list, you should be fine. The old fortran compiler should be just fine, I'd guess.

Reply:
Ah my bad, the error specifically mentions that it's a CGAL error, so it can't be a gfortran issue. But that also means that I have tried everything and it has not worked.

Reply:
This is a modern project with a minimum Python of 3.6 (no 2.7). I doubt the C++ requires a bleeding edge compiler. We've been manually triggering these test-deployments in the GitHub UI already. Are you talking about needing to manually trigger when we're ready to release 3.3.4.0rc1? (It would be better to trigger that through a tag/release for the sake of updating to rc2, rc3, etc.)

Reply:
You are stuck with whatever comes on manylinux anyway. This is 2010, so GCC 8 IIRC. The fortran will either already be included (so GCC 8 based) or you will get it from apt, which will then be GCC 4.6 based or something equally as ridiculously old (remember, manylinux2010 is CentOS 6 based).

What error are you looking at?

Reply:
Manually triggering to see if this PR passes. Currently it only builds the tests and not the wheels, so you have to manually trigger GHA to see if the PR can be built into wheels.

Reply:
(which it looks like you are doing, when I look into actions history)

Reply:
```
checking for CGAL/Exact_predicates_inexact_constructions_kernel.h... no
checking CGAL... no
```
This looks familiar, did we not solve it before?

Reply:
You could try manylinux2014, just to see if CentOS 7 is better, or you could even do manylinux_2_24, which is Debian based but very "new", in linux and pip that are supported, but has an older compiler (GCC 6 IIRC). You'll need the Ubuntu commands for that one.